TAC/Meetings/2025/2025-05-15/CCC_TAC_Minutes-2025-05-15

```diff
@@ -0,0 +1,118 @@
+* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
+* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)
+
+## Agenda and Minutes
+
+### Welcome
+
+* Dan Middleton (DM) opened the call at 7:03 am PT
+* DM welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation
+* Renu Chauhan (RC) & Riaan Kleinhans (RK) created meeting minutes.
+* DM reviewed the agenda
+
+
+### Roll Call / Attendance
+
+A quorum were not required
+
+#### Voting members of the TAC
+
+*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
+
+* AMD - Nathaniel McCallum +   / David Kaplan
+* Arm - Paul Howard  
+* Google - Catherine Zhang  +  / Catalin Sandu 
+* Huawei - Zhipeng (Howard) Huang 
+* Intel - Dan Middleton (TAC chair) +  / Simon Johnson
+* Meta - Henry Wang / Kevin Hui
+* Microsoft - Alec Fernandez +
+* Nvidia - Fritz Alder +
+* Red Hat -  Yash Mankad +   / Ram Pai 
+* TikTok -  Mingshen Sun  / Yao Zhang
+* Shielded Technologies - Bob  Blessing-Hartley 
+
+   " + " indicates attending
+
+#### Other Attendees
+
+* Abigail Halder (Mainsail Industries)
+* Bokdeuk Jeong (Samsung)
+* Chris Ramming  
+* Derek Miller
+* Jiewen Yao (Intel)
+* John Suykerbuyk
+* Kanth Ghatraju (Oracle)
+* Manu Fontaine (Hushmesh)
+* Mark Novak (JP Morgan Chase)
+* Renu Chauhan (Linux Foundation)
+* Riaan Kleinhans (Linux Foundation)
+
+
+## Welcome New Community Members
+ Abigail Halder (AH), John Suykerbuyk (JS) introduced themselves.
+
+## Meeting Minute Approval & Old Business
+* Meeting minutes will be approved via GitHub. DM reviewed the old Business covered in the previous meeting. 
+
+## Announcements
+* SIG Repos
+DM informed about the creation of two repositories under the governance organisation.  
+
+* TAC Tech Talk Guidelines up (Thanks, Fritz!)
+DM expressed gratitude to Fritz Alder for creating the TAC Tech Talk guidelines & which can be referred to in future. 
+
+* CCC-Wide Security Disclosure Process
+DM discussed implementing a security disclosure process for the CCC community.
+
+## Annual Project Review: SPDM-Tools 
+Yao, Jiewen, presented the annual project review for the SPDM-Tools project.
+
+## CCC LF mentorship: Islet  
+Bokdeuk Jeong (BJ) shared the CCC LF mentorship experience.
+
+## Veracruz: 
+Derek Miller shared the latest updates for the Veracruz project and the latest progress on its movement to the emeritus state. Discussion ensued.
+
+##  Live Review GRC Patterns Docs
+DM requested feedback on the GRC pattern Documents. Discussion ensued about the open PR and change required before merging.
+
+## Topic Schedule 2025  
+* DM reviewed the schedule for the upcoming weeks.
+
+## Projects
+
+## SIGs
```

Comment:
Do we normally have these sections in the minutes or are these extraneous?

Reply:
We pre-populate the headings to simplify note taking. Deleted.